foundation/application.go

```diff
@@ -178,6 +179,10 @@ func setEnv() {
 			support.Env = support.EnvArtisan
 		}
 	}
+
+	env := flag.String("env", ".env", "custom .env path")
+	flag.Parse()
+	support.EnvPath = *env
```

Comment:
CI failed, we may need to add the code below here:
```
var env *string
if support.Env == support.EnvTest {
  testEnv := ".env"
  env = &testEnv
} else {
  env = flag.String("env", ".env", "custom .env path")
  flag.Parse()
}
```

Reply:
> CI failed, we may need to add the code below here:
> 
> ```
> var env *string
> if support.Env == support.EnvTest {
>   testEnv := ".env"
>   env = &testEnv
> } else {
>   env = flag.String("env", ".env", "custom .env path")
>   flag.Parse()
> }
> ```

The `flag.Parse()` of the flag package only allow calls once in the application. It seems that there are also calls in `testify`, and repeated calls make it impossible to parse.